app/backend/src/clj/teet/project/project_commands.clj

```diff
@@ -28,14 +30,23 @@
               :thk.project/custom-end-m]
           project-eid))
 
+(defn update-vektorio-project-name? [db project-id project-name]
```

Comment:
The function ending in ? implies that this returns a boolean and doesn't do any side-effects.